### PR TITLE
Improvements to standalone tests

### PR DESCRIPTION
**Pull request by dzafman**
Fixes: https://tracker.ceph.com/issues/39162

- [x] Add qa/tasks/divergent_priors2.py
- [x] Create new test cases
- [x] References tracker ticket
- [x] ~Updates documentation if necessary~
- [x] ~Includes tests for new functionality or reproducer for bug~

### CODE REVIEW

**Comment by dzafman** [comment]
Testing passed
http://pulpito.ceph.com/dzafman-2019-04-10_11:40:17-rados-wip-zafman-testing-distro-basic-smithi/

File: qa/standalone/osd/osd-rep-recov-eio.sh [hunk 1]
```diff
@@ -44,9 +44,10 @@ function run() {
 function setup_osds() {
     local count=$1
     shift
+    local type=$1
```
Comment (tchaikov) [inline qa/standalone/osd/osd-rep-recov-eio.sh]: shall we default to `bluestore` here? like
```sh
local type =${2:-_bluestore}
```

Comment (dzafman) [reply qa/standalone/osd/osd-rep-recov-eio.sh]: @tchaikov No, there is no longer a run_osd_bluestore().  Normally, we use run_osd() which currently defaults to bluestone.  There is now a run_osd_filestore() as a special type.

**Comment by dzafman** [comment]
@neha-ojha See commit comment for 7e778980016fcac80ca30df081d85df415865139

**Comment by neha-ojha** [comment]
> @neha-ojha See commit comment for [7e77898](https://github.com/ceph/ceph/commit/7e778980016fcac80ca30df081d85df415865139)

looks good

**Pull request merged**